src/Parser.js

```diff
@@ -757,6 +757,9 @@ Parser.prototype.parseOptionalGroup = function() {
     }
 };
 
+var cjkRegex =
+    /[\u3040-\u309F]|[\u30A0-\u30FF]|[\u4E00-\u9FAF]|[\uAC00-\uD7AF]/;
+
```

Comment:
Where did those ranges come from? Adding that information as a comment might be useful if we have to extend the ranges at some point.